src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1334,6 +1344,36 @@ static ucs_status_t uct_dc_mlx5_calc_sq_length(uct_ib_mlx5_md_t *md,
     return UCS_OK;
 }
 
+static void
+uct_dc_mlx5_dci_set_port_affinity(uct_dc_mlx5_iface_t *iface,
+                                  const uct_dc_mlx5_iface_config_t *config)
+{
+    uct_ib_mlx5_md_t *md = ucs_derived_of(iface->super.super.super.super.md,
+                                          uct_ib_mlx5_md_t);
+
+    iface->tx.port_affinity = 0;
+    if (md->super.dev.port_select_mode == UCT_IB_LAG_QUEUE_AFFINITY ||
+        md->super.dev.port_select_mode == UCT_IB_LAG_PORT_SELECT_FT) {
+        switch (config->tx_port_affinity) {
+        case UCS_CONFIG_ON:
+            iface->tx.port_affinity = 1;
+            break;
+        case UCS_CONFIG_AUTO:
+            if (md->super.dev.port_select_mode == UCT_IB_LAG_QUEUE_AFFINITY) {
+                iface->tx.port_affinity = 1;
+            }
+            break;
+        default:
+            break;
+        }
+    } else {
+        if (config->tx_port_affinity == UCS_CONFIG_ON) {
+            ucs_warn("not support set LAG_PORT_AFFINITY in dev: %s",
+                     uct_ib_device_name(&md->super.dev));
```

Comment:
Device %s does not support UCX_IB_DC_LAG_PORT_AFFINITY=on, port select mode is %s
(need to check full correct env var name)

Reply:
@evgeny-leksikov Done. Please help review it. I used ```port select mode is %d``` instead of ```port select mode is %s```, or it needs to define new enum-string mapping. It's enough for developers to get the port select mode value.